conda/recipes/libcuml/meta.yaml

```diff
@@ -67,21 +85,31 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
       run:
-        - cudatoolkit {{ cuda_spec }}
-        - libcublas {{ libcublas_run_version }}
-        - libcublas-dev {{ libcublas_run_version }}
-        - libcufft {{ libcufft_run_version }}
+        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        - libcublas {{ cuda11_libcublas_run_version }}
+        - libcublas-dev {{ cuda11_libcublas_run_version }}
+        - libcufft {{ cuda11_libcufft_run_version }}
+        - libcurand {{ cuda11_libcurand_run_version }}
+        - libcusparse {{ cuda11_libcusparse_run_version }}
+        - libcusparse-dev {{ cuda11_libcusparse_run_version }}
+        {% else %}
+        - cuda-cudart-dev
+        - libcublas-dev
+        - libcufft
+        - libcurand
+        - libcusparse
```

Comment:
This question also seems related to some of what I've asked about above.

Reply:
dev packages were not needed indeed, left by inertia from some older versions (where we might have been over-provisioning them tbh)

Reply:
To follow up on the previous conversation with some history, I think these `-dev` packages were needed when RAFT wasn't yet migrated to GitHub Actions. I think that would have imposed the requirement that cuml needed `-dev` variants. [`494dc8f` (#5075)](https://github.com/rapidsai/cuml/pull/5075/commits/494dc8fe24802357fbe5182d0919389448650b39)

This is no longer an issue, and I believe this PR is correct now.

Reply:
Are we missing `cusolver` here? It looks like libcuml uses libcusolver directly. It may be supplied transitively by RAFT which is why we aren't seeing failures, but we should list it explicitly.
```suggestion
        - libcusolver
        - libcusparse
```